runtimerevolution/web3-roulette
# [Web3-30] Giveaway countdown card

# Description

This PR adds the giveaway countdown card. It only shows up if the logged in user is registered in any giveaway that will end in less than 100 hours. If more than one satisfies these conditions then the nearest one to end is chosen.

Ticket: https://runtime-revolution.atlassian.net/browse/WEB3-30

## Type of change

- New feature (non-breaking change which adds functionality)

## How Has This Been Tested?

1. Create a new giveaway with end date in less than 100 hours
2. Participate in the giveaway
3. Go to the giveaways list page
4. Verify that the countdown card shows up

## Checklist:
